locales/en-US/pocket.properties

```diff
@@ -26,6 +26,8 @@ tagline = Save articles and videos from Firefox to view in Pocket on any device,
 taglinestory_one = Click the Pocket Button to save any article, video or page from Firefox.
 taglinestory_two = View in Pocket on any device, any time.
 tagssaved = Tags Added
+tos = By continuing, you agree to Pocket's *Terms of Service* and *Privacy Policy*
```

Comment:
I advise against doing this, keep the markup to make people understand they're links, just avoid exposing the URL. Not sure what kind of replacement you plan to do, but it's really brittle, unless you plan to double check all localizations.